salesagility/SuiteCRM
# Fix #10120 - Inbound Email Issues

## Description





## Motivation and Context


## How To Test This

1. Go to Inbound Emails
2. You should now not be able to save the record with no type.
3. Testing Connection should no longer hang/ timeout
4. 
## Types of changes


### Final checklist
